ga4gh/backend.py

```diff
@@ -708,3 +729,90 @@ def __init__(self, dataDir):
             dataset = datasets.FileSystemDataset(datasetDir)
             self._datasetIdMap[dataset.getId()] = dataset
         self._datasetIds = sorted(self._datasetIdMap.keys())
+
+
+class GeneotypePhenotypeBackend(AbstractBackend):
+    """
+    an rdf object store
+    """
+    # keep the graph as a static variable
+    def __init__(self, is_prod):
+        super(GeneotypePhenotypeBackend, self).__init__()
+        self.dataset = genotype_phenotype.G2PDataset(is_prod)
+
+
+class GeneotypePhenotypeIterator(IntervalIterator):
```

Comment:
@jeromekelleher  Hi.  I'm looking at making this change. As you point out, the GeneotypePhenotypeIterator  class is large compared to what it actually needs to do.  The tradeoff seems to be between  keeping the class and allowing G2P requests to run via the existing `runSearchRequest` vs creating my own implementation of runSearchRequest.  I'm completely open to that possibility, just wanted to verify that it made sense to you.